cli/flags/common.go

```diff
@@ -70,6 +71,8 @@ func (commonOpts *CommonOptions) InstallFlags(flags *pflag.FlagSet) {
 	// opts.ValidateHost is not used here, so as to allow connection helpers
 	hostOpt := opts.NewNamedListOptsRef("hosts", &commonOpts.Hosts, nil)
 	flags.VarP(hostOpt, "host", "H", "Daemon socket(s) to connect to")
+	flags.StringVarP(&commonOpts.Context, "context", "c", "",
```

Comment:
Needs a follow-up to update the completion scripts (and the documentation / man pages)

/cc @albers

Reply:
I plan to add documentation as part of #1501. I might need help for completion scripts though.